gradle.properties

```diff
@@ -19,9 +19,9 @@ buildKotlinVersion=1.3.0-rc-116
 buildKotlinCompilerRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_130_CompilerAllPlugins),number:1.3.0-rc-116,tag:kotlin-native,pinned:true/artifacts/content/maven
 remoteRoot=konan_tests
 testDataVersion=1226829:id
-kotlinCompilerRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_dev_CompilerAllPlugins),number:1.3.20-dev-1253,tag:kotlin-native,pinned:true/artifacts/content/maven
-kotlinVersion=1.3.20-dev-1253
-testKotlinVersion=1.3.20-dev-1253
+kotlinCompilerRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_dev_CompilerAllPlugins),number:1.3.20-dev-1403,branch:default:any/artifacts/content/maven
+kotlinVersion=1.3.20-dev-1403
+testKotlinVersion=1.3.20-dev-1403
```

Comment:
The versions are updated to get the preview of new declarations and tests from a branch. Do not merge until it gets to master.